### DISCUSSION TITLE
Why is forums a group???

### DISCUSSION THREAD

**Discussion by Secret-chest** [ScratchAddons/ScratchAddons#8046] [Discussion]
Groups are for sorting addons based on their size. Categories are for filtering addons based on the part of the site they affect. Why is Forums, then, a group, and not a website subcategory?

**Comment by mxmou** [comment]
Groups are based on usefulness, not size. Forums is a separate group because

> Forum addons are only useful for a small subset of Scratch users. That's why they're hidden by default, and they can only be hidden if they're in a separate group (or there would be no easily discoverable way to show them).

https://github.com/ScratchAddons/ScratchAddons/issues/2617#issuecomment-1052468963

**Comment by Samq64** [comment]
Forums is also a cubcategory. It's not perfect but it's the way it's almost always been.

**Reply by WorldLanguages** [reply]
It's a subcategory so that those searching for forums addons can easily find them.
It's also a group so that it's collapsed by default, since the average user won't care about recommended forum addons, or featured forum addons.

**Comment by Secret-chest** [comment]
Then we can have a more general Extras subgroup, but limiting to forums is weird.

**Reply by DNin01** [reply]
We try to organize addons to benefit the user base as a whole, and many Scratchers don't use the forums, which is also a very different area of the Scratch website, that often. To avoid cluttering the list for these users, we section forum addons off into their own group to bring more attention to addons for the rest of Scratch.

The group's name could be changed to something more general like "extras" but then the group would lose its meaning and probably look even weirder.

A solution to this would be a filter system with forum addons unchecked by default.

**Comment by iqnite** [comment]
It's been confusing me since the beginning, too. My proposal in https://github.com/ScratchAddons/ScratchAddons/discussions/7712 would kinda fix that.

**Reply by DNin01** [reply]
See above (https://github.com/ScratchAddons/ScratchAddons/discussions/8046#discussioncomment-11720065).

I guess the Forums group can seem a little strange if you don't know what its purpose is.

**Reply by iqnite** [reply]
I know, that's why reorganizing the settings would erase the need for this, as the Forum addons would be hidden in a subcategory anyways.

**Reply by DNin01** [reply]
> I know, that's why reorganizing the settings would erase the need for this, as the Forum addons would be hidden in a subcategory anyways. 

I'm not sure about hiding it from an "All" section.

**Reply by mxmou** [reply]
> I'm not sure about hiding it from an "All" section.

There wouldn't be an "All" section if #7712 was implemented.
